### ISSUE TITLE
Error message is only shown after pressing CTRL-C in Windows

### ISSUE THREAD

**Issue by strets123** [conda/conda-build#1712]
When trying to build without git installed then conda build freezes and only displays an error after CTRL-C

There should instead be a message before the user presses ctrl-c

**Comment by msarahan** [comment]
Please post your ``conda info`` output

**Comment by strets123** [comment]
(C:\Users\astretton\Miniconda3) E:\>conda info
Current conda install:

               platform : win-64
          conda version : 4.2.13
       conda is private : False
      conda-env version : 4.2.13
    conda-build version : 2.1.2
         python version : 3.5.2.final.0
       requests version : 2.12.4
       root environment : C:\Users\astretton\Miniconda3  (writable)
    default environment : C:\Users\astretton\Miniconda3
       envs directories : C:\Users\astretton\Miniconda3\envs
          package cache : C:\Users\astretton\Miniconda3\pkgs
           channel URLs : https://conda.anaconda.org/conda-forge/win-64
                          https://conda.anaconda.org/conda-forge/noarch
                          https://repo.continuum.io/pkgs/free/win-64
                          https://repo.continuum.io/pkgs/free/noarch
                          https://repo.continuum.io/pkgs/pro/win-64
                          https://repo.continuum.io/pkgs/pro/noarch
                          https://repo.continuum.io/pkgs/msys2/win-64
                          https://repo.continuum.io/pkgs/msys2/noarch
            config file : C:\Users\astretton\.condarc
           offline mode : False

**Comment by msarahan** [comment]
Thanks.  I'll try to isolate where this hang is happening - it might be trying to install git behind the scenes since you don't already have it in place.  There is code for that that gets triggered when the recipe needs git but it is not available on the system.  How long have you waited?  Git takes a long time to install on Windows...

**Comment by strets123** [comment]
I waited at least 5 minutes...

Ah OK, this makes sense. What I have found is that things work fine for me if I am running windows server and logged in as an administrator account.

If I install Conda globally and use the administrator command prompt then it also works fine. I don't want to do this however because of issues it causes with permissions and shared folders. 

Maybe the approach could be to add code to elevate if needed, perhaps by creating a separate nullsoft installer which runs the git installer a bit like how conda constructor installers do it? 

At a minimum there could be a message saying what the script is trying to do.
